oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/session/OakJcrConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.jackrabbit.oak.jcr.session;
+
+public interface OakJcrConstants {
+    int DEFAULT_WARN_LOG_STRING_SIZE_THRESHOLD_VALUE = 102400;
```

Comment:
I don't recall our earlier discussion, but I though we were saying that indexing end (OAK-8642) would log for 100K while jcr side would be a bit more tolerable. But maybe, I'm mis-remembering.

Reply:
AFAIR, two different values were not suggested. If you think we should have different values, let me know what should be default value for jcr.

Reply:
Nah... It's just what I recalled. I'm on both ways.